stripe/util.py

```diff
@@ -199,7 +202,19 @@ def convert_to_dict(obj):
         return obj
 
 
-def populate_headers(idempotency_key):
+@overload
+def populate_headers(
+    idempotency_key: str,
+) -> "dict[Literal['Idempotency-Key'], str]":
```

Comment:
`typing_extensions.Dict`?

Reply:
Fixed with [Use Dict not "dict[...]"](https://github.com/stripe/stripe-python/pull/1020/commits/ab5f1e9547fe7e4b7aeab51c6ff6d845a284f1af)